middle.mkd

```diff
@@ -285,7 +288,7 @@ processing software SHALL NOT be expected to have access to coordinate
 reference systems databases.  Applications requiring CRS other than the
 default MUST assume all responsibility for reference system and
 coordinate accuracy. Furthermore, GeoJSON coordinates MUST NOT under any
-circumstances use latitude, longitude order. See Section 6,
+circumstances use latitude, longitude order. See Section 8,
```

Comment:
Yep. I'm trying internal references in 88dfe6f. Excerpt from output in `draft.txt`:

```
   o  A GeoJSON object MAY have a "bbox" member, the value of which MUST
      be a bounding box array (see Section 5).

   o  A GeoJSON object MAY have any number of other members (see
      Section 6).
```